swp/forms/publication.py

```diff
@@ -64,6 +66,11 @@ def clean_authors(self) -> Sequence[str]:
 
         return [clean(author) for author in items]
 
+    def clean_publication_dt(self) -> datetime.date:
+        value = self.truncated_field('publication_date')
+        default = f'{datetime.date.today().year}-01-01'
```

Comment:
When working with dates and datetimes in django consider using the timezone aware utils from `django.utils.timezone`.